tl/cli/feature-voting.py

```diff
@@ -0,0 +1,68 @@
+import sys
+import traceback
+import argparse
+
+from tl.exceptions import TLException
+
+
+def parser():
+    return {
+        'help': """
+        Score candidates using feature voting. List of features may be used: 
+        1. exact match top 1: using pre-computed embedding vectors, either from a file or from elasticsearch.
+        2. page rank 
+        3. qnode with smallest number
+        4. Weiss Waterman distance
+        5. Jaccard between description and row
+        6. column query (SemTab 2020 paper)
+        """
+    }
+
+
+def add_arguments(parser):
+    # input file
+    parser.add_argument('input_file', nargs='?', type=argparse.FileType('r'), default=sys.stdin)
+
+    # Features used in voting
+    parser.add_argument(
+        '-c', '--input-column-names', action='store', dest='input_column_names',
+        default='qnode_score',
+        help="the column name of features to use in voting, separated by ','"
```

Comment:
should add options here

Reply:
user will not choose from given option, they can specify any numeric value column as feature